lab_1/main.py

```diff
@@ -2,8 +2,9 @@
 Lab 1
 A concordance extraction
 """
+text = open('data.txt', 'r', encoding= 'utf-8').read()
```

Comment:
remove this line. You do not need to open your text here. Just write required functions